### PR TITLE
DPP-1308 add copy to deds mapping

### PR DESCRIPTION
**Pull request by scottywakefield**
- Removed a bunch of files that should have been ignored
- Added rule to .gitignore to avoid future recurrences
- Updated existing mappings file
- Fixed typo in Transient Payments tbl.

### CODE REVIEW

File: src/PeriodEnd/TransferPayments/DeployTransferPayments/copy mappings/DasTransfersCopyToDedsMapping.xml [hunk 1]
```diff
@@ -9,49 +9,60 @@
...
         <ColumnMapping>
-          <Source>CommitmentId</Source>
-          <Destination>CommitmentId</Destination>
+          <Source>SendingAccountId</Source>
+          <Destination>SendingAccountId</Destination>
         </ColumnMapping>
         <ColumnMapping>
-          <Source>CommitmentVersionId</Source>
-          <Destination>CommitmentVersionId</Destination>
+          <Source>RecievingAccountId</Source>
```
Comment (davehowlett) [inline src/PeriodEnd/TransferPayments/DeployTransferPayments/copy mappings/DasTransfersCopyToDedsMapping.xml]: typo - should be receiving

File: src/PeriodEnd/TransferPayments/DeployTransferPayments/copy mappings/DasTransfersCopyToDedsMapping.xml [hunk 2]
```diff
@@ -9,49 +9,60 @@
...
+          <Destination>RecievingAccountId</Destination>
```
Comment (davehowlett) [inline src/PeriodEnd/TransferPayments/DeployTransferPayments/copy mappings/DasTransfersCopyToDedsMapping.xml]: typo - should be receiving

**Review by davehowlett — CHANGES_REQUESTED** [review]
Need to fix the typos in the mapping files - I've checked the table definition for TransferPayments.AccountTransfers and it is spelled correctly (ReceivingaccountId)

**Comment by scottywakefield** [comment]
@davehowlett fixed that, please check again.

**Review by davehowlett — APPROVED** [review]
looks ok now

**Pull request merged**